test/320_claim_3_test.sh

```diff
@@ -13,12 +14,11 @@ delete_persistence() {
 
 start_suite "Claiming addresses"
 
-weave_on $HOST1 launch-router --ipalloc-range $UNIVERSE
+weave_on $HOST1 launch-router --ipalloc-range $UNIVERSE $HOST2
```

Comment:
without this change, host1 can form a ring without talking to host2, but I want to test what happens when it has to wait for consensus.